cypress/integration/common/web_steps.js

```diff
@@ -1,13 +1,65 @@
-import { Then, Given } from 'cypress-cucumber-preprocessor/steps'
+import { Then, Given, When } from 'cypress-cucumber-preprocessor/steps'
 
-Then(`I should see {string}`, (textToFind) => {
+Then(`I should see {string}`, textToFind => {
   cy.contains(textToFind)
 })
 
-Given(`I am at the {string} page`, (pageName) => {
+Given(`I am at the {string} page`, pageName => {
   if (pageName === 'Home') {
     cy.visit('/')
   } else {
-    cy.fail('unknown page \'' + pageName + '\'')
+    cy.visit('/' + pageName)
   }
 })
+
+Given('I am at the projects page', () => {
+  cy.server()
+  cy.fixture('projects').then(projects => {
+    cy.route(
+      /\/api\/v1\/projects/,
+      projects
+    ).as('getProjects')
+    cy.visit('/projects')
+    cy.window()
+      .its('store')
+      .invoke('dispatch', { type: 'GET_PROJECTS', payload: projects })
+  })
+  cy.wait('@getProjects')
+})
+
+When('I click on the LocalSupport project', () => {
+  cy.server()
```

Comment:
I would prefer this to be in a Background scenario

Reply:
when you say this, do you mean `cy.server()`... and by Background scenario you mean like the cucumber background where we would say something like `Given the server has started`?